doc/_admin-guide/070_Destinations/270_sql/003_sql_destination_options.md

```diff
@@ -231,7 +231,7 @@ following sections.
 ## values()
 
 |  Type:  |    string list|
-|Default: |  \"\${R\_YEAR}-\${R\_MONTH}-\${R\_DAY}, \${R\_HOUR}:\${R\_MIN}:\${R\_SEC}\", \"\${FACILITY}\", \"\${LEVEL}\", \"\${HOST}\", \"\${PROGRAM}\", \"\${PID}\", \"\${MSGONLY}\"|
+|Default: |  \"${R\_YEAR}-${R\_MONTH}-${R\_DAY}, ${R\_HOUR}:${R\_MIN}:${R\_SEC}\", \"${FACILITY}\", \"${LEVEL}\", \"${HOST}\", \"${PROGRAM}\", \"${PID}\", \"${MSGONLY}\"|
```

Comment:
maybe the table is just again the reason, but probably the other escapings are not needed either?!